src/webui/index.js

```diff
@@ -39,12 +39,12 @@ const createWindow = () => {
     window.webContents.openDevTools()
   }
 
-  window.webContents.on('crashed', event => {
-    logger.error(`[web ui] crashed: ${event.toString()}`)
+  window.webContents.on('render-process-gone', (_, { reason, exitCode }) => {
+    logger.error(`[web ui] crashed: ${reason}, code: ${exitCode}`)
   })
 
-  window.webContents.on('unresponsive', event => {
-    logger.error(`[web ui] unresponsive: ${event.toString()}`)
+  window.webContents.on('unresponsive', () => {
+    logger.error('[web ui] the webui became unresponsive')
```

Comment:
Would it be possible to add a dialog here, where user can decide if they want to kill a rogue webui  process before it slows everything down to a crawl for some reason?

Found [a poc here](https://www.electronjs.org/docs/latest/api/web-contents#contentsforcefullycrashrenderer) (just needs i18n strings):

```
contents.on('unresponsive', async () => {
  const { response } = await dialog.showMessageBox({
    message: 'App X has become unresponsive',
    title: 'Do you want to try forcefully reloading the app?',
    buttons: ['OK', 'Cancel'],
    cancelId: 1
  })
  if (response === 0) {
    contents.forcefullyCrashRenderer()
    contents.reload()
  }
})
```